### ISSUE TITLE
Update ForeignKeyRef class with names

### ISSUE THREAD

**Issue by robes** [informatics-isi-edu/ermrestjs#106]
The `ForeignKeyRef` needs additional properties:
```
ForeignKeyRef
  .constraint_names -> from the schema returned by ermrest
  .to_name -> an annotation on the foreign key
  .from_name -> an annotation on the foreign key
```

This is a pre-requisite for #102 .

**Comment by robes** [comment]
See this for the FKR names:
https://github.com/informatics-isi-edu/ermrest/blob/master/api-doc/model/rest.md#foreign-key-creation

Note the caveat that it is actually (I forgot) a list of names because FKRs can have more than one name.

**Comment by RFSH** [comment]
It's implemented as part of #122.
